test/test_grid.py

```diff
@@ -0,0 +1,85 @@
+import os
+import numpy as np
+import xarray as xr
+
+from unittest import TestCase
+from pathlib import Path
+
+import uxarray as ux
+
+
+class TestGrid(TestCase):
+
+    def test_saveas(self):
+        """Rename a file."""
+        current_path = Path(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
For the "current_path" variable definitions here and the one in test_dataset, I believe we can just move them outside of the testing functions and reduce to just one at the top of the file. I did this in my PR for test_scrip and it worked, but I haven't had the chance to try in your PR. If you want me to (and there isn't another reason not to do so), I'd gladly test and implement this change on my local of the branch and push the changes!

Reply:
Sure, please test and push. Thanks!